ignite/distributed/auto.py

```diff
@@ -73,6 +109,8 @@ def auto_dataloader(dataset, **kwargs):
             kwargs["num_workers"] = (kwargs["num_workers"] + nproc - 1) // nproc
 
         if "batch_sampler" not in kwargs:
+            sampler: Union[DistributedProxySampler, DistributedSampler]
```

Comment:
We support py3.5 and unfortunately inline type hints like that wont work.

Reply:
Thanks, good to know! Will fix it.